### PR TITLE
fix(prowlarr): simplify breaking nginx config

### PR DESCRIPTION
**Pull request by flying-sausages**
## Description
Simplifies prowlarr nginx config

## Fixes issues: 
afbeelding
🤷 

## Proposed Changes:
Couldn't reproduce the issue, but it wouldn't hurt to simplify the config down anyway

## Change Categories

- Bug fix 
## Checklist

- [x] Branch was made off the `develop` branch and the PR is targetting the `develop` branch
- [x] Docs have been made OR are not necessary
    - PR link: 
- [x] Changes to panel have been made OR are not necessary
    - PR link: 
- [x] Code is formatted [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Code conforms to project structure [(See more)](https://swizzin.ltd/dev/structure)
- [x] Shellcheck isn't screaming [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Prints to terminal are handled [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#printing-into-the-terminal)
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] Testing was done
   - [ ] Tests created or no new tests necessary
   - [ ] Tests executed

### Architectures

|   			| `amd64` 	| `armhf` 	| `arm64` 	| Unspecified 	|
|--------		|-------- 	|-------- 	|-------- 	|----------		|
| Focal 		|			|			|			|				|
| Bionic		|			|			|			|				|
| Buster		|			|			|			|				|
| Stretch		|			|			|			|				|
| Raspbian  	|	⚫️		|			|	⚫️		|	⚫️			|

### CODE REVIEW

**Comment by bakerboy448** [comment]
tested the new nginx; does not work at all

the below DOES work
```
location /prowlarr {
  proxy_pass        http://127.0.0.1:9696/prowlarr;
  proxy_set_header Host $host;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header X-Forwarded-Host $host;
  proxy_set_header X-Forwarded-Proto https;
  proxy_redirect off;

  proxy_http_version 1.1;
  proxy_set_header Upgrade $http_upgrade;
  proxy_set_header Connection $http_connection;
}

  location /prowlarr/api { auth_request off;
  proxy_pass       http://127.0.0.1:9696/prowlarr/api;
}

  location ~ /prowlarr/[0-9]+/api { auth_request off;
  proxy_pass       http://127.0.0.1:9696;
}

  location /prowlarr/Content { auth_request off;
    proxy_pass http://127.0.0.1:9696/prowlarr/Content;
 }
```
 https://wiki.servarr.com/prowlarr/installation#nginx

**Comment by bakerboy448** [comment]
will this be merged anytime soon to stop providing users with bad installs?
@brettpetch ?

File: scripts/nginx/prowlarr.sh [hunk 1]
```diff
@@ -17,33 +17,38 @@ user="$PROWLARR_OWNER"
...
-  proxy_pass        http://127.0.0.1:$app_port/$app_baseurl;
-  proxy_set_header Host \$proxy_host;
-  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
-  proxy_set_header X-Forwarded-Proto \$scheme;
-  proxy_redirect off;
-  auth_basic "What's the password?";
-  auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
+    proxy_pass          http://127.0.0.1:$app_port/$app_baseurl;
+    proxy_http_version  1.1;
+    proxy_set_header    Host                \$proxy_host;
```
Comment (bakerboy448) [inline scripts/nginx/prowlarr.sh:26]: this needs to be $host

**Comment by bakerboy448** [comment]
https://github.com/swizzin/swizzin/issues/796

File: scripts/nginx/prowlarr.sh [hunk 2]
```diff
@@ -17,33 +17,38 @@ user="$PROWLARR_OWNER"
...
+    proxy_set_header    X-Forwarded-For     \$proxy_add_x_forwarded_for;
+    proxy_set_header    X-Forwarded-Proto   \$scheme;
```
Comment (bakerboy448) [inline scripts/nginx/prowlarr.sh:28]: missing header
`  proxy_set_header X-Forwarded-Host  $host;`

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
see comments; the conf is still wrong and not what FS was told to change it to.

**Review by brettpetch — APPROVED** [review]

**Comment by brettpetch** [comment]
@bakerboy448 can you please confirm that this is up to date? If so I'll merge into develop ;b

File: scripts/nginx/prowlarr.sh [hunk 3]
```diff
@@ -17,33 +17,39 @@ user="$PROWLARR_OWNER"
...
-  proxy_set_header Connection \$http_connection;
- }
-  # Allow the App API
-  location /$app_baseurl/api { auth_request off;
-    proxy_pass http://127.0.0.1:$app_port/$app_baseurl/api;
- }
-  # Allow Content
-  location /$app_baseurl/Content { auth_request off;
-    proxy_pass http://127.0.0.1:$app_port/$app_baseurl/Content;
+# Allow Content
```
Comment (bakerboy448) [inline scripts/nginx/prowlarr.sh:43]: Can be removed

File: scripts/nginx/prowlarr.sh [hunk 4]
```diff
@@ -17,33 +17,39 @@ user="$PROWLARR_OWNER"
...
+    proxy_set_header    Connection          \$http_connection;
+    proxy_redirect      off;
+    auth_basic              "What's the password?";
+    auth_basic_user_file    /etc/htpasswd.d/htpasswd.${master};
+}
+
+# Allow the App API
+location /$app_baseurl/api {
+    auth_request    off;
+    proxy_pass      http://127.0.0.1:$app_port/$app_baseurl/api;
```
Comment (bakerboy448) [inline scripts/nginx/prowlarr.sh:40]: This can just be IP:Port, no?

Compare to SWAGs confs

**Comment by liaralabs** [comment]
Can this get closed as part of the recent *arr nginx proxy PR?

**Pull request closed without merging**